src/types/Options.ts

```diff
@@ -1,9 +1,15 @@
 import { DefaultEnvVars, UndefinedEnvVars } from './EnvVars';
 import { RemoveKeys } from './helpers';
 
+export interface EnvOptions {
+  pathToEnv?: string;
+  envFileName?: string;
+}
+
 export interface Options<Required extends UndefinedEnvVars = DefaultEnvVars, Optional extends UndefinedEnvVars = DefaultEnvVars> {
   required?: Required;
   optional?: RemoveKeys<Optional, keyof Required>;
+  dotEnvOptions?: EnvOptions;
```

Comment:
Let's just make this `options?: Options` so we can use it for more things going forward